packages/langium/CHANGELOG.md

```diff
@@ -1,5 +1,67 @@
 # Change Log of `langium`
 
+## v2.0.0 (Aug. 2023)
+
+### EcmaScript Modules (ESM)
+
+Langium is now compiling to ESM instead of CommonJS (CJS).
+CommonJS is the JavaScript convention introduced by Node.js and has historically been used in all Node.js based application.
+With the introduction of ESM to modern versions of the Node.js runtime, more and more projects and libraries are making the move to ESM. Since a lot of our dependencies have moved on, we decided to do so as well.
+
+While ESM based projects in Node.js can continue to import CJS code, CJS projects cannot (easily) import ESM code.
+This is a **very important change** for adopters, as it prevents using Langium as-is in CJS projects.
+This leaves us with a problem, as all vscode extensions need to run as CJS code.
+Luckily, JavaScript bundlers such as `esbuild` are capable of transforming ESM code into CJS code.
+Using a bundler for vscode extensions and language servers is heavily recommended anyway,
+so we hope that adopters don't have much trouble upgrading to Langium 2.0.
```

Comment:
Just a note. It may be worth the effort to add some documentation or a tutorial to support migration specifically from Langium 1.3.0 to 2.0.0, possibly using a yeoman generator example language as a basis.